stage0/run.go

```diff
@@ -280,8 +275,17 @@ func generatePodManifest(cfg PrepareConfig, dir string) ([]byte, error) {
 			ra.App.Group = group
 		}
 
-		if cfg.InheritEnv || len(cfg.ExplicitEnv) > 0 {
-			MergeEnvs(&ra.App.Environment, cfg.InheritEnv, cfg.ExplicitEnv)
+		// loading the environment from the lowest priority to highest
+		if cfg.InheritEnv {
+			// Inherit environment does not override app image environment
+			mergeEnvs(&ra.App.Environment, os.Environ(), false)
+		}
+
+		if len(cfg.EnvFromFile) > 0 {
+			mergeEnvs(&ra.App.Environment, cfg.EnvFromFile, true)
```

Comment:
if "len(cfg.EnvFromFile) == 0", mergeEnvs is a no-op so the "if" is not necessary.

Ditto below.